packages/client/src/runtime/query.ts

```diff
@@ -610,6 +615,10 @@ function stringify(value: any, inputType?: DMMF.SchemaArgInputType) {
     return JSON.stringify(value.toString('base64'))
   }
 
+  if (value instanceof FieldRefImpl) {
+    return `{ _ref: ${JSON.stringify(value.name)}}`
```

Comment:
Do we have a form of validation at runtime or is it type-level only?

Reply:
It's both run- and compile-time.